Scripts/VerifyNoBuildSettings.swift

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@ElektrojungeMS @clpolet Is it okay to add copyright to this file?

Reply:
@jaelim-ms Yeah, initially it's based on 3rd party, but now the most part of it is my changes/code